src/features/query-bar/components/fixtures.js

```diff
@@ -5,13 +5,41 @@ const initialCondition: ConditionType = {
     id: 0,
     columnDisplayText: '',
     columnKey: null,
+    fieldId: '3',
```

Comment:
This is odd. I don't see ConditionType.fieldId in flowTypes.

I see this in the Condition-test as well. What am I missing?